app/grants/clr.py

```diff
@@ -95,8 +102,8 @@ def translate_data(grants_data):
 '''
 def get_verified_list(grant_contributions):
     verified_list = []
-    for proj, user, ver_stat, amount in grant_contributions:
-        if ver_stat == 'verified' and user not in verified_list:
+    for _, user, ver_stat, _ in grant_contributions:
+        if ver_stat and user not in verified_list:
```

Comment:
hm, why do we need if `ver_stat` not in `verified_list` if the list is primarily users?

Reply:
I'm not clear here ! Isn't that what you intended when you wrote it up?

Reply:
sorry wasn't thinking right on the ver_stat variable. it's boolean now, not a string. 👍